models/levels.js

```diff
@@ -27,7 +27,6 @@ const addLevel = async (levelData) => {
 const deleteLevel = async (id) => {
   try {
     await levelModel.doc(id).delete();
-    return;
```

Comment:
why are we removing this.

Reply:
it is because of lint fix, it's removing this return statement